crates/driver/src/infra/config/file/mod.rs

```diff
@@ -363,6 +363,8 @@ enum UniswapV3Config {
         /// How many pools to initialize during start up.
         #[serde(default = "uniswap_v3::default_max_pools_to_initialize")]
         max_pools_to_initialize: usize,
+
+        graph_api_base_url: Url,
```

Comment:
Can we move the four occurrences here (crates/driver/src/infra/config/file/mod.rs) to a single field on the top level `LiquidityConfig` object? Since we are not going to configure different base URLs for different liquidity types, this seems the least configuration overhead.